doc/sphinx/addendum/extraction.rst

```diff
@@ -313,14 +313,21 @@ The system also provides a mechanism to specify ML terms for inductive
 types and constructors. For instance, the user may want to use the ML
 native boolean type instead of the |Coq| one. The syntax is the following:
 
-.. cmd:: Extract Inductive @qualid => @string [ {+ @string } ]
+.. cmd:: Extract Inductive @qualid => @string__1 [ {+ @string } ]
 
```

Comment:
Is this change of name (string -> string__1) really intended? or some automatic renaming gone wild?

Reply:
I did it so that I could refer to that particular string in the description that comes just after. It is used in some other places of the manual (the `1` will appear as a subscript).